libs/sdk-backend-bear/src/backend/organization/securitySettings.ts

```diff
@@ -0,0 +1,35 @@
+// (C) 2021 GoodData Corporation
+
+import { ISecuritySettingsService, ValidationContext } from "@gooddata/sdk-backend-spi";
+import { SDK } from "@gooddata/api-client-bear";
+
+export interface IValidationResponse {
+    validationResponse: {
+        valid: boolean;
+    };
+}
+
+export class SecuritySettingsService implements ISecuritySettingsService {
+    /**
+     * Constructs a new SecuritySettingsService
+     * @param sdk - SDK for making a backend calls
+     * @param scope - URI of the scope. For now only the organization (domain) URI is supported by the backend. The plan is to support also workspace URI and user profile URI.
+     */
+    constructor(private readonly sdk: SDK, public readonly scope: string) {}
```

Comment:
need authApiCall guard here instead of SDK instance (see above)

Reply:
Fixed.